jpo-ode-svcs/src/main/java/us/dot/its/jpo/ode/traveler/TravelerMessageBuilder.java

```diff
@@ -1,188 +1,206 @@
-//        } else if ("Speed Limit".equals(contentType)) {
-//            content.setSpeedLimit(buildSpeedLimit(codes));
-//        } else if ("Exit Service".equals(contentType)) {
-//            content.setExitService(buildExitService(codes));
-//        } else {
-//            content.setGenericSign(buildGenericSignage(codes));
-//        }
-//        return content;
-//    }
-//
-//    private ITIScodesAndText buildAdvisory(String[] codes) {
-//        ITIScodesAndText itisText = new ITIScodesAndText();
-//        for (String code: codes) {
-//            ITIScodesAndText.Sequence_ seq = new ITIScodesAndText.Sequence_();
-//            ITIScodesAndText.Sequence_.Item item = new ITIScodesAndText.Sequence_.Item();
-//            item.setItis(Long.parseLong(code));
-//            seq.setItem(item);
-//            itisText.add(seq);
-//        }
-//        return itisText;
-//    }
-//
-//    private WorkZone buildWorkZone(String[] codes) {
-//        WorkZone wz = new WorkZone();
-//        for (String code: codes) {
-//            WorkZone.Sequence_ seq = new WorkZone.Sequence_();
-//            WorkZone.Sequence_.Item item = new WorkZone.Sequence_.Item();
-//            item.setItis(Long.parseLong(code));
-//            seq.setItem(item);
-//            wz.add(seq);
-//        }
-//
-//        return wz;
-//    }
-//
-//    private SpeedLimit buildSpeedLimit(String[] codes) {
-//        SpeedLimit sl = new SpeedLimit();
-//        for (String code: codes) {
-//            SpeedLimit.Sequence_ seq = new SpeedLimit.Sequence_();
-//            SpeedLimit.Sequence_.Item item = new SpeedLimit.Sequence_.Item();
-//            item.setItis(Long.parseLong(code));
-//            seq.setItem(item);
-//            sl.add(seq);
-//        }
-//        return sl;
-//    }
-//
-//    private ExitService buildExitService(String[] codes) {
-//        ExitService es = new ExitService();
-//        for (String code: codes) {
-//            ExitService.Sequence_ seq = new ExitService.Sequence_();
-//            ExitService.Sequence_.Item item = new ExitService.Sequence_.Item();
-//            item.setItis(Long.parseLong(code));
-//            seq.setItem(item);
-//            es.add(seq);
-//        }
-//        return es;
+package us.dot.its.jpo.ode.traveler;
+
+
+import com.oss.asn1.EncodeFailedException;
+import com.oss.asn1.EncodeNotSupportedException;
+import us.dot.its.jpo.ode.j2735.dsrc.*;
+import us.dot.its.jpo.ode.j2735.dsrc.TravelerDataFrame.Content;
+import us.dot.its.jpo.ode.j2735.dsrc.TravelerDataFrame.MsgId;
+import us.dot.its.jpo.ode.j2735.itis.ITIScodesAndText;
+
+import java.nio.ByteBuffer;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Calendar;
+import java.util.Date;
+
+
+
+
+
+/**
+ * Created by anthonychen on 2/16/17.
+ */
+public class TravelerMessageBuilder {
+    public TravelerInformation travelerInfo;
+    private static final SimpleDateFormat sdf = new SimpleDateFormat("MM/dd/yyyy HH:mm a");
```

Comment:
Use DateTimeUtils in jpo-ode-common. It provides utilities for doing ISO format.